src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -0,0 +1,299 @@
+import glob
+import os
+
+from natsort import natsorted
+
+from swsssdk import ConfigDBConnector
+from swsssdk import SonicDBConfig
+from swsssdk import SonicV2Connector
+
+from .device_info import CONTAINER_PLATFORM_PATH
+from .device_info import HOST_DEVICE_PATH
+from .device_info import get_platform
+
+ASIC_NAME_PREFIX = "asic"
+NAMESPACE_PATH_GLOB = "/run/netns/*"
+ASIC_CONF_FILENAME = "asic.conf"
+FRONTEND_ASIC_SUB_ROLE = "FrontEnd"
+BACKEND_ASIC_SUB_ROLE = "BackEnd"
+EXTERNAL_PORT = 'Ext'
+INTERNAL_PORT = 'Int'
+PORT_CHANNEL_CFG_DB_TABLE = 'PORTCHANNEL'
+PORT_CFG_DB_TABLE = 'PORT'
+BGP_NEIGH_CFG_DB_TABLE = 'BGP_NEIGHBOR'
+NEIGH_DEVICE_METADATA_CFG_DB_TABLE = "DEVICE_NEIGHBOR_METADATA"
+DEFAULT_NAMESPACE = ''
+PORT_ROLE = 'role'
```

Comment:
This file uses a mixture of `'` and `"` for strings and docstrings. Please pick one and use it consistently throughout the file.

Reply:
Fixed in the latest commit